<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlasmaTransfusionGUI</class>
 <widget class="QMainWindow" name="PlasmaTransfusionGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlasmaTransfusion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QLineEdit" name="txtInputDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>79</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Age Directory:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInputPath">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOldAgeName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Old Age name:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnConvertAge">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>370</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert Age</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QLineEdit" name="txtOutputDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Directory:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOutputPath">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNewAgeName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>129</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>New Age name (optional):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNewSequencePrefix">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>New Sequence prefix (optional):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cbVersion">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>160</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Plasma version:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
